src/cryptoadvance/specter/templates/includes/addresses-table.html

```diff
@@ -0,0 +1,488 @@
+    </style>
+
+    <nav class="row">
+        <button type="button" class="receive-addresses-view-btn btn radio left checked">
+            Receive Addresses
+        </button>
+        <button type="button" class="change-addresses-view-btn btn radio right">
+            Change Addresses
+        </button>
+    </nav>
+
+    <div class="pagination-container">
+        <a class="pagination-back pagination-arrow pagination-disabled">&#8249;</a>
+        Page <input class="pagination-idx" type="number" step="1" min="1" value="1"></input><span
+            class="pagination-counter"></span>
+        <a class="pagination-next pagination-arrow pagination-disabled">&#8250;</a>
+        <br>
+    </div>
+    <span class="page-limit-label">Page limit: </span>
+    <fieldset class="page-limit-fieldset">
+        <select class="page-limit-select">
+            <option value="10">10</option>
+            <option value="25">25</option>
+            <option value="50" selected>50</option>
+            <option value="100">100</option>
+            <option value="0">All</option>
+        </select>
+    </fieldset>
+
+    <br>
+    <div class="export-container hidden">
+        <h1>Export addresses to CSV</h1>
+
+
+
+        <div>
+            <span class="switch-text"></span>
+            <label class="switch">
+                <input type="checkbox" class="only-current-type-checkbox" checked>
+                <span class="slider"></span>
+            </label><br><br>
+        </div>
+
+
+
+        <a class="export btn" download>Export</a>
+    </div>
+
+    <table class="addresses-table">
+
+        <thead>
+            <tr>
+                <th value="index" class="index-header">
+                    <div class="index-arrow down-arrow"></div>
+                </th>
+                <th value="address" class="address-header">Addresses<div class="address-arrow"></div>
+                </th>
+                <th value="label" class="label-header">Label<div class="label-arrow">
+                </th>
+                <th value="used" class="used-header">Used<div class="used-arrow">
+                </th>
+                <th value="utxo" class="utxo-header">UTXO<div class="utxo-arrow">
+                </th>
+                <th value="amount" class="amount-header">Amount<div class="amount-arrow">
+                </th>
+                <th style="width: 50px; padding: 10px;">
+                    <button type="button" class="export-btn btn" style="width: 50px;">Export</button>
+                </th>
+            </tr>
+        </thead>
+
+        <tr class="empty">
+            <td></td>
+            <td>Fetching addresses ...</td>
+            <td class="optional"></td>
+            <td class="optional"></td>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+
+        <tbody class="addr-tbody">
+        </tbody>
+    </table>
+</template>
+
+<script type="text/javascript">
+    class AddrTableElement extends HTMLElement {
```

Comment:
Same would be cool for classes in general:
* What is this class
* Where is it used? How is it used?
* How does it behave?

Reply:
It seems that the constructor is somehow magically extracting arguments from the existing page. Maybe it refers to a different class? So then this should be added to the documentation as well.

Reply:
Ok. I've added the description of this class.